app/views/warehouse_reports/outflow/_filter.haml

```diff
@@ -7,31 +7,36 @@
         = f.input :end, label: 'End Date', as: :date_picker, input_html: {value: @filter.end}
```

Comment:
I reorganized the filters a bunch to make it cleaner (hopefully). Does this organization and labeling make sense @eanders ? Also might be jarring for folks if they're used to filters being in certain places. Happy to move it back if you think.

<img width="1315" alt="Screen Shot 2022-12-09 at 11 24 31 AM" src="https://user-images.githubusercontent.com/5333982/206747125-e6557011-7708-4707-8159-a5b8afadf246.png">

Reply:
Also I changed `No Project Limit` -> `No Additional Projects` so it matches the label.  But looking at the code, it looks like it actually is a limit, not "additional projects" being added..
https://github.com/greenriver/hmis-warehouse/blob/8e81d7a48a0b708dfe5382b43793511bc5ee33d1/app/models/grda_warehouse/warehouse_reports/outflow_report.rb#L227-L247

Should it be "Projects _excluded_ from the "without recent service" calculation.?

Reply:
That's "make sure clients don't have service in these projects" right?
The re-work of the filters looks great.  Thanks!

Reply:
Yeah, I guess that's what it's doing. Should I change the wording, or does that placeholder text make sense?

Reply:
Let's label it:
"Additional Projects used to exclude clients when checking "without recent service"
and the placeholder can stay as you have it.
Does that read ok to you?

Reply:
Yeah, I think that's better. Updated. thanks